analyzers/src/SonarAnalyzer.CSharp/Rules/PropertiesShouldBePreferred.cs

```diff
@@ -48,10 +48,10 @@ public sealed class PropertiesShouldBePreferred : SonarDiagnosticAnalyzer
 
                     foreach (var candidate in propertyCandidates)
                     {
-                        c.ReportIssue(Diagnostic.Create(
+                        c.ReportIssue(
                             Rule,
                             candidate.Locations.FirstOrDefault(),
-                            messageArgs: candidate.Name));
+                            messageArgs: candidate.Name);
```

Comment:
Can go away. And as you'll open the file, the invocation can be a single line
```suggestion
                            candidate.Name);
```